src/lib/importExport.ts

```diff
@@ -7,11 +7,21 @@ import {
 } from 'expo-file-system'
 import * as Sharing from 'expo-sharing'
 import { Alert, Platform } from 'react-native'
+import CryptoJS from 'react-native-crypto-js'
 
 import { Secret } from '../types'
 
 const mimeType = 'application/octet-stream'
 
+function isJsonString(str) {
+  try {
+    JSON.parse(str)
+  } catch (e) {
+    return false
+  }
+  return true
```

Comment:
it would be more idiomatic to see this done as try { return true } catch { return false }